### DISCUSSION TITLE
Pydantic's Mkdocs Linking to Auto-generated Documentation

### DISCUSSION THREAD

**Discussion by thomasmarwitz** [pydantic/pydantic#10124] [Question] [answered]
I would like to understand what setting or plugin Pydantic uses to be able to abbreviate links to auto-generated API documentation.

E.g. in `docs/concepts/models.md`, there is just `pydantic.main.BaseModel.model_rebuild` used to reference the `model_rebuild` function. 

On the website however, 
<img width="756" alt="image" src="https://github.com/user-attachments/assets/5a2cf23f-8f23-4551-8d96-b45308260c77">

this is translated into `api/base_model/#pydantic.BaseModel.model_rebuild`.

**Comment by Viicos** [comment, marked as answer]
We use `mkdocstrings` to do so: https://mkdocstrings.github.io/usage/#cross-references

**Reply by thomasmarwitz** [reply]
Thanks!
